bsip-0075.md

```diff
@@ -0,0 +1,47 @@
+```
+BSIP: 0075
+Title: Asset Owner Defines MCR and MSSR Values
+Authors:
+  Bangzi1001
+  John Jones <jmjatlanta@gmail.com>
+Status: Draft
+Type: Protocol
+Created: 2019-09-19
+Discussion: https://github.com/bitshares/bsips/issues/96
+Worker: TBD
+```
+
+# Abstract
+This BSIP provides the ability for the asset creator to decide if adjustments to the Maintenance Collateral Ratio (MCR) or the Maximum Short Squeeze Ratio (MSSR) should be done by the asset owner or by price feed providers.
+
+# Motivation
+Asset owners are often in the best position to control the values of MCR and MSSR of their asset. Therefore, they should be able to have direct control to adjust those values, should they choose to allow themselves.
+
+# Rationale
+Prior to this change, adjustments to MCR and MSSR were handled by feed producers. This often requires the asset owner to contact the feed producer and ask for the change. For some assets, such a step is time consuming for both parties and ill-fitted as part of the feed producer's responsibilities.
+
+This new change shall permit the asset owner to determine *at asset creation time* who should have the responsibility to maintain those values.
```

Comment:
How will existing smartcoins be affected?

Reply:
Existing smartcoins aren't affected. The owners can change settings after the protocol upgrade though.